src/Forms/FormTextArea.test.js

```diff
@@ -39,4 +49,87 @@ describe('<FormTextArea />', () => {
         expect(ref.current.tagName).toEqual('TEXTAREA');
         expect(ref.current.className).toEqual('fd-textarea');
     });
+
+    describe('FormTextArea counter', () => {
+        const setup = (props, children) => {
+            return mount(<FormTextarea {...props}>
+                {children}
```

Comment:
I see warnings running these tests
```
    Warning: Use the `defaultValue` or `value` props instead of setting children on <textarea>.
        in textarea (created by FormTextarea)
        in FormTextarea (created by WrapperComponent)
        in WrapperComponent
```
it could be changed to

```jsx
        const setup = (props, defaultValue) => {
            return mount(<FormTextarea {...props} defaultValue={defaultValue} />)
        }
```
to get rid of the warnings